.php_cs.dist

```diff
@@ -28,7 +28,7 @@ return PhpCsFixer\Config::create()
         'no_null_property_initialization' => true,
         'no_short_echo_tag' => true,
         'no_unreachable_default_argument_value' => true,
-        'no_useless_else' => true,
+        'no_useless_else' => ['convert_elseif_to_if' => true],
```

Comment:
So we would consider this a bugfix? I'm ok with that, though I'll have to target 2.2 then :p